pkg/generator/helper/testdata/TestProcessMetaFiles/custom_step_code_generated.golden

```diff
@@ -200,6 +201,27 @@ func testStepMetadata() config.StepData {
 					},
 				},
 			},
+			Outputs: config.StepOutputs{
+				Resources: []config.StepResources{
+					{
+						Name: "commonPipelineEnvironment",
+						Type: "piperEnvironment",
+						Parameters: []map[string]interface{}{
+							{"Name": "artifactVersion"},
+							{"Name": "git/commitId"},
+							{"Name": "git/branch"},
+							{"Name": "custom/customList"},
+						},
+					},
+					{
+						Name: "influxTest",
+						Type: "influx",
+						Parameters: []map[string]interface{}{
```

Comment:
do we have a chance to better specify the type?

Reply:
even less chance, because it *is* a nested type that contains maps as values (as visible in the line below)

Reply:
ok, thanks